slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/SlashingProtectionContext.java

```diff
@@ -12,22 +12,34 @@
  */
 package tech.pegasys.web3signer.slashingprotection;
 
+import java.util.Optional;
+
 import org.jdbi.v3.core.Jdbi;
 
 public class SlashingProtectionContext {
 
   private final Jdbi slashingProtectionJdbi;
-  private final Jdbi pruningJdbi;
+  private final Optional<SlashingProtectionPruner> slashingProtectionPruner;
```

Comment:
nit: don't see the value in this being Optional since you're not using the empty case for anything. It'll error when you call .get() regardless.

Reply:
Removed the Optional